Source/ACE.Server/Entity/Landblock.cs

```diff
@@ -219,8 +219,10 @@ private void SpawnEncounters()
 
                 if (wo == null) continue;
 
-                var xPos = Math.Clamp(encounter.CellX * 24.0f, 0.5f, 191.5f);
-                var yPos = Math.Clamp(encounter.CellY * 24.0f, 0.5f, 191.5f);
+                var radius = wo.PhysicsObj.GetRadius();
```

Comment:
This throws a nullref exception.

Would the PhysicsObj ever be init at this point?

Reply:
hmm, i thought this at first too, but from testing, all of the encounters spawned correctly, without throwing null exception...

also testing patch live now, and oddly enough, it hasn't thrown a null exception yet..

looking at the logic, it looks like it should be immediately..

Reply:
oh i see whats happening...

it actually is getting an invisible null exception, that gets swallowed up and not displayed for some reason (possibly because this is all running in a separate Task)

Reply:
fixed